src/m365/aad/commands/app/app-add.ts

```diff
@@ -567,6 +588,24 @@ class AadAppAddCommand extends GraphCommand {
         value: password.secretText
       }));
   }
+  
+  private getCertificateBase64Encoded(args: CommandArgs, logger: Logger): Promise<string | undefined> {
```

Comment:
Let's align it with the same method in `aad app set` and make it sync as there's no reason to return a Promise

Reply:
Right, I could've seen that myself. Changed it!